src/backends/interflop-mca/interflop_mca.c

```diff
@@ -54,30 +54,39 @@
 #include <unistd.h>
 
 #include "../../common/float_const.h"
+#include "../../common/float_struct.h"
 #include "../../common/interflop.h"
 #include "../../common/tinymt64.h"
 
-#include "quadmath-imp.h"
+typedef enum {
+  KEY_PREC_B32,
+  KEY_PREC_B64,
+  KEY_MODE = 'm',
+  KEY_SEED = 's'
+} key_args;
 
 typedef struct {
-  int choose_seed;
+  bool choose_seed;
   uint64_t seed;
 } t_context;
 
 /* define the available MCA modes of operation */
-#define MCAMODE_IEEE 0
-#define MCAMODE_MCA 1
-#define MCAMODE_PB 2
-#define MCAMODE_RR 3
+typedef enum { ieee, mca, pb, rr } mcamode;
```

Comment:
IMHO I found the older names more readable (MCAMODE_xx) I don't care about using enum or define variable, but it would be easier to read to explicitly expose that we are talking about MCAMODE and not about some local variable with lowercase names.

Reply:
I think that `enum` is better since

> Enumerations permit the declaration of named constants in a more convenient and structured fashion than does #define; they are visible in the debugger, obey scope rules, and participate in the type system.

but I agree with the readability.